apps/api/src/cart/cart.controller.ts

```diff
@@ -30,6 +31,14 @@ export class CartController {
     return cart;
   }
 
+  @Get('/')
+  async getCart(@Request() req) {
+    const userID = await this.cartService.extractIdFromToken(req);
+    const userCart = await this.cartService.getCart(userID);
+    if (!userCart) throw new NotFoundException('Cart is empty');
```

Comment:
move the condition for the exception in service to allow handling of this error when the function is re-used.